ui/main.qml

```diff
@@ -361,6 +371,7 @@ ApplicationWindow {
         id: existingKey
         ExistingKey {
             onClosed: function () {
+                appSettings.showBackupMessage = true
```

Comment:
If it is set to true by default, I think this one here is not necessary right?

Reply:
It can theoretically happen that you first start importing an existing key but then decide to stop the process and generate a new key.
If we don't reset the property to `true` then by that point it'll be `false`. If you then generate a key and perform the login, it'll cause the app to remove the mnemonic from the db without the user having a chance to back it up.